subgraph/subgraph.yaml

```diff
@@ -109,5 +110,8 @@ templates:
         - event: LockMetadata(string,string,string)
           handler: handleLockMetadata
           receipt: true
+        - event: ReferrerFee(indexed address,uint256)
```

Comment:
Here is the event we will be listening to in all of the evm blockchains that `Unlock Protocol` supports